code/game/objects/structures/arclumin.dm

```diff
@@ -8,11 +8,14 @@ but we are all coders together.
 
 Goodbye, man. We'll miss you.
 
+This memorial has been designed for him and any future coders to perish.
```

Comment:
Grammatically, the wording here implies this is placed here so that coders can perish. It should be written "any future coders _who_ perish" and that implies it is here if that happens.